### PR TITLE
[DOC-2732] Added examples of expressions

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 1]
```diff
@@ -336,17 +336,42 @@ Whether the number in a variable is treated as a double or string depends on the
 
 If you enter 123 in a string filed, such as a name, it is treated as a string. If you enter 123 in a count field, such as instance count, it is treated as a double.
 
+### Contains
+
+When using `contains`, make sure the expression is wrapped within `<+ >` and the specific string is within `"`.
+
+For example, `<+stage.name.contains(“s1”)>`.
```
Comment (SudarshanPurohit) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:343]: Can we make sure the straight vertical " is used in the examples, so they can be copied and pasted? I think the slanted ones will not work.

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 2]
```diff
@@ -336,17 +336,42 @@ Whether the number in a variable is treated as a double or string depends on the
...
+When using ternary conditional `?:` operators, do not use spaces between the operators and values. Make sure the expression is wrapped within `<+ >`.
+
+For example, `<+condition ? <value_if_true> : <value_if_false>>` will not work. Use `<+condition?<value_if_true>:<value_if_false>>` instead.
+
+### Equals
+
+When using `==` condition, make sure the expression is wrapped within `<+ >`.
 
-For example, `condition ? <value_if_true> : <value_if_false>` will not work. Use `condition?<value_if_true>:<value_if_false>` instead.
+For example, `<+<+pipeline.name> == “pipeline1“>`.
```
Comment (SudarshanPurohit) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:368]: Here, too, the straight double quotes are needed.

Comment (SudarshanPurohit) [reply docs/platform/12_Variables-and-Expressions/harness-variables.md:368]: Also, can we have a better example here, @archit-harness ? I assume this expression is evaluated to a true or false - is that the intention, to print the true/false? Perhaps an example to use the value to evaluate something? 

btw, is the output the string true/false, or 0/1?

**Review by SudarshanPurohit — COMMENTED** [review]
A question for @archit-harness here to verify.

File: docs/platform/12_Variables-and-Expressions/harness-variables.md (hunk 1 above)
Comment (rashminandasahoo) [reply docs/platform/12_Variables-and-Expressions/harness-variables.md:343]: Done

**Review by archit-harness — DISMISSED** [review]

**Review by SudarshanPurohit — APPROVED** [review]

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 3]
```diff
@@ -336,17 +336,42 @@ Whether the number in a variable is treated as a double or string depends on the
 
 If you enter 123 in a string filed, such as a name, it is treated as a string. If you enter 123 in a count field, such as instance count, it is treated as a double.
 
+### Contains
+
+When using `contains`, make sure the expression is wrapped within `<+ >` and the specific string is within `"`.
+
```
Comment (SmritiSatya) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:342]: Can we use 'ensure' instead of 'make sure'? (global)

**Review by SmritiSatya — APPROVED** [review]

**Review by SmritiSatya — DISMISSED** [review]

**Pull request merged**